src/state/apis/snapshot/snapshot.ts

```diff
@@ -121,7 +165,7 @@ export const snapshotApi = createApi({
             FEE_CURVE_PARAMETERS[model].FEE_CURVE_FOX_DISCOUNT_DELAY_HOURS,
           )
           const delegation = false // don't let people delegate for discounts - ambiguous in spec
-          const votingPowerResults = await Promise.all(
+          const votingPowerResults = await Promise.allSettled(
```

Comment:
I don't think we want to change this tbh. The idea was that if we can't fetch a users voting power correctly and in a complete manner, that is on us and we default to free trades since we can't make a judgement. This change would potentially result in someone that should have the correct voting power to achieve free trades, to be charged depending on what may or may not have been fetched.

Reply:
as per dms, I forgot we changed default to max bps so best effort is actually better for user now. disregard, this makes sense.